bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "health-facilities",
+  "version": "1.0.0",
+  "dependencies": {
+    "agrc": "agrc-widgets#4.0.0",
```

Comment:
the latest version of this is 5 if you want the latest basemap nice to haves that scott added.